source/includes/kubernetes/_k8_persistentvolumes.md

```diff
@@ -0,0 +1,178 @@
+## Persistent volumes
+
+<!-------------------- LIST Persistent volumes -------------------->
+
+#### List persistent volumes
+
+```shell
+curl -X GET \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/a_service/an_environment/persistentvolumes"
+```
+
+> The above command returns a JSON structured like this:
+
+```json
+{
+  "data": [
+    {
+      "id": "pvc-05097a93-120d-45d2-aaab-0f273849fccd",
+      "metadata": {
+        "annotations": {
+          "pv.kubernetes.io/provisioned-by": "rook-ceph.rbd.csi.ceph.com"
+        },
+        "creationTimestamp": "2020-07-03T09:12:17.000-04:00",
+        "finalizers": [
+          "kubernetes.io/pv-protection"
+        ],
+        "name": "pvc-05097a93-120d-45d2-aaab-0f273849fccd",
+        "resourceVersion": "147569183",
+        "selfLink": "/api/v1/persistentvolumes/pvc-05097a93-120d-45d2-aaab-0f273849fccd",
+        "uid": "2e20a8fb-d90c-4c39-acd8-7007f85e2d8e"
+      },
+      "spec": {
+        "accessModes": [
+          "ReadWriteOnce"
+        ],
+        "capacity": {
+          "storage": "8Gi"
+        },
+        "claimRef": {
+          "apiVersion": "v1",
+          "kind": "PersistentVolumeClaim",
+          "name": "data-cmc-cluster-zookeeper-1",
+          "namespace": "cmc-kafka",
+          "resourceVersion": "147569124",
+          "uid": "05097a93-120d-45d2-aaab-0f273849fccd"
+        },
+        "csi": {
+          "driver": "rook-ceph.rbd.csi.ceph.com",
+          "fsType": "ext4",
+          "nodeStageSecretRef": {
+            "name": "rook-csi-rbd-node",
+            "namespace": "rook-ceph"
+          },
+          "volumeAttributes": {
+            "clusterID": "rook-ceph",
+            "imageFeatures": "layering",
+            "imageFormat": "2",
+            "pool": "replicapool",
+            "storage.kubernetes.io/csiProvisionerIdentity": "1593101040137-8081-rook-ceph.rbd.csi.ceph.com"
+          },
+          "volumeHandle": "0001-0009-rook-ceph-0000000000000001-d11d2a4d-bd2e-11ea-b56e-7a7db811cab1"
+        },
+        "persistentVolumeReclaimPolicy": "Delete",
+        "storageClassName": "rook-ceph-block",
+        "volumeMode": "Filesystem"
+      },
+      "status": {
+        "phase": "Bound"
+      }
+    }
+    ],
+  "metadata": {
+    "recordCount": 1
+  }
+```
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/persistentvolumes</code>
```

Comment:
Do we have a syntax error here? The diff appears in red on my end

Reply:
Same, I don't know why but it compiles and run file locally

Reply:
c8669a717950a28af3a98b2b044b2a1374c282ed